### PR TITLE
Update introduction language to introduce and relate VC language closer to OIDC language.

### PR DESCRIPTION
**Pull request by AdamJLemmon**
@tplooker Have a read through the updated language here.
I agree it would be great to begin to introduce the common VC language into the OIDC community and maintain such through this document.
I have updated the introduction to be quite clear and explicit how the new terminology directly relates to existing OIDC language.
If ok with this language then I can continue through and make PRs to update the other sections accordingly.
Let me know what you think, thanks!

### CODE REVIEW

**Comment by quartzjer** [comment]
@AdamJLemmon this is excellent! You're describing it in language that comes through very crisply to me which I genuinely appreciate.

Only thing that might make it even more clear for readers would be expanding Holder (OP) and Holder (RP) to Holder (acting as an OP) and Holder (acting as an RP).

I also like your simple definition of a credential in this context, it is just a set of claims bound to a holder in a cryptographically verifiable way.

**Comment by AdamJLemmon** [comment]
Thanks @quartzjer ! really appreciate the review and feedback :)
I like it, that additional language certainly helps to improve readability.
Update incoming!

File: spec.md [hunk 1]
```diff
@@ -62,9 +62,15 @@ This specification defines how an OpenID provider can be extended beyond being t
...
 
-In typical deployments of OpenID Connect today to be able to exercise the identity an End-User has with an OpenID Provider with a relying party, the relying party must be in direct contact with the provider. This constraint causes issues such as  [relying party tracking](https://github.com/WICG/WebID#the-rp-tracking-problem).
+In typical deployments of OpenID Connect today, in order for an OpenID Provider (OP) to be able to exercise the identity an End-User has with a Relying Party (RP), the RP must be in direct contact with the OP. This constraint causes issues such as [RP tracking](https://github.com/WICG/WebID#the-rp-tracking-problem).
+
+This specification defines how an OP can be extended to provide a set of claims to a RP of which that RP controls and may present (now acting as an OP) onward to other RPs.  This specification will introduce the roles of a Credential Issuer (OP), a Holder (RP and OP) and a Verifier (RP).
```
Comment (tplooker) [inline spec.md:67]:
```suggestion
This specification defines how an OP can be extended to provide a set of claims to an RP of which that RP controls and may present (now acting as an OP) onward to other RPs.  This specification will introduce the roles of a Credential Issuer (OP), a Holder (RP and OP) and a Verifier (RP).
```

**Comment by tplooker** [comment]
Hey @AdamJLemmon these are great improvements, made one quick suggestion but other than that, happy to merge

**Comment by AdamJLemmon** [comment]
Perfect thanks @tplooker good for me!
Once merged I'll proceed to do some editorial changes through out to reflect this language :)

**Pull request merged**